modules/data.atmosphere/R/download.NARR_site.R

```diff
@@ -269,7 +269,7 @@ get_NARR_thredds <- function(start_date, end_date, lat.in, lon.in,
           url,
           robustly(get_narr_url, n = 20, timeout = 1),
           xy = xy,
-          flx = FALSE,
+          .data$flx = FALSE,
```

Comment:
```suggestion
          flx = FALSE,
```

Reply:
@infotroph if we have a constant value at the right hand side of the argument (flx = constant) will flx here need ' .data' pronoun.

Reply:
@moki1202 I don't believe `.data` is ever allowed on the left no matter what's on the right side. 

But the key tricky bit here is that `flx = FALSE` gets parsed as an argument to `map` which is then passed on as an argument to `get_narr_url`,  not as an argument to `mutate`. The nesting is kind of confusing, but the overall effect is approximately equivalent to this loop (I'll ignore `robustly`, which is a wrapper that repeatedly calls `get_narr_url` until it succeeds):

```
xy <- "a lat/lon value defined outside the dataframe" 

sfc_data_raw <- sfc_df
sfc_data_raw$data <- NULL
for (i in seq_along(sfc_df$url)) {
  sfc_data_raw$data[[i]] <- get_narr_url(sfc_df$url[[i]], xy = xy, flx = FALSE)
}
```